### PR TITLE
fix(mailservice): mailService fix subject

### PR DESCRIPTION
**Pull request by Zeroks77**
Hey Guys, here is my fix for the MailService,

there was just a wrong '&' instead of a '?'

This is the result of the & 
#422 

And here is the fix 

image

this also solves my problems in #382

### CODE REVIEW

**Comment by mschwrdtnr** [comment]
Please change the pr-title like our commit format guidelines.

File: Phonebook.Frontend/.gitignore [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
-key.pem
-cert.pem
-
-# e2e
-/e2e/*.js
-/e2e/*.map
-
-# System Files
-.DS_Store
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (mschwrdtnr) [inline Phonebook.Frontend/.gitignore]: don't commit stuff like that.

Comment (Zeroks77) [reply Phonebook.Frontend/.gitignore]: i dont understand why, but i had to commit the .gitignore without any changes. 

VsCode and powershell show that the file is modified. 

Any idea why git does this ?

Comment (DanielHabenicht) [reply Phonebook.Frontend/.gitignore]: line endings

Comment (Zeroks77) [reply Phonebook.Frontend/.gitignore]: hopefully fixed

Comment (DanielHabenicht) [reply Phonebook.Frontend/.gitignore]: thats the last time, after this it should be fixed

File: .gitattributes [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Handle line endings automatically for files detected as text
 # and leave all files detected as binary untouched.
 # uses the optimal lineending but prioritize lf
-* text=auto eol=lf
+* text=input
```
Comment (paule96) [inline .gitattributes]: put this in extra PR.

Comment (Zeroks77) [reply .gitattributes]: :P

**Comment by DanielHabenicht** [comment]
Gosh, why does Github not update the DIFF?

**Comment by Zeroks77** [comment]
🤷‍♂️

**Comment by DanielHabenicht** [comment]
@Zeroks77 can you cherry pick https://github.com/T-Systems-MMS/phonebook/pull/423/commits/8a3a84afb2c53a77a1376c1730c6056c46b93bcb onto another branch and make a PR?

**Comment by Zeroks77** [comment]
sure

**Comment by mschwrdtnr** [comment]
reopened in #427

**Pull request closed without merging**